addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4917,6 +4917,16 @@ registry.Box = SnippetOptionWidget.extend({
 
 
 registry.layout_column = SnippetOptionWidget.extend({
+    /**
+     * @override
+     */
+    cleanForSave() {
+        // Remove the padding highlights.
+        const highlightedEl = this.$target[0].querySelector('.o_we_padding_highlight');
+        if (highlightedEl) {
```

Comment:
I'll do `this.$target[0].querySelectorAll('.o_we_padding_highlight')).forEach(...)` instead :) 
Since it is apparently better than using for loops (?)

Reply:
I do not know about "better"... but in this case I'd say it is a matter of taste. 
For huge loops where performance would matter, I think the `forEach` overhead (due to the 2 additional layers of calls) makes the keyword-based `for ... of` loop preferable.